index.test.js

```diff
@@ -154,7 +154,7 @@ test("plugin will hash sourcemap with includeMap", () => {
             map: { inline: false },
         })
         .then(result => {
-            const expectedOutput = '.a {}\n/*# sourceMappingURL=file01.88aab.css.map */';
+            const expectedOutput = '.a {}\n/*# sourceMappingURL=file01.b5686.css.map */';
```

Comment:
As far as I can tell, I had to update this hash in the tests because of source map changes in PostCSS 8. We hash the source map in the plugin, so the hash changes.